docs/api.md

```diff
@@ -2141,6 +2146,48 @@ will be returned.
   <!--end-response-body-get-passwordforgotstatus-ttl-->
 
 
+### Recovery codes
+
+#### GET /recoveryCodes
+
+:lock: HAWK-authenticated with session token
+<!--begin-route-get-recoverycodes-->
+
+<!--end-route-get-recoverycodes-->
+
+##### Response body
+
+* `recoveryCodes`: *array, items(string, regex(validators.HEX_STRING))*
+
+  <!--begin-response-body-get-recoverycodes-recoveryCodes-->
+  
+  <!--end-response-body-get-recoverycodes-recoveryCodes-->
+
+
+#### POST /session/verify/recoveryCode
+
+:lock: HAWK-authenticated with session token
+<!--begin-route-post-sessionverifyrecoverycode-->
+
```

Comment:
I think we should try to add a sentence or two describing the new routes in the docs, even if their usage seems obvious to us from the names.

Reply:
...I'm still not seeing a description for the new endpoints? 😄